.npmignore

```diff
@@ -0,0 +1,8 @@
+NodeReport*.txt
```

Comment:
Yes, npm always includes certain files (sorry, I can't get any more specific of a link than [this](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)). I'm only throwing it out there as a suggestion because it is slightly more future proof. I don't have a problem with the approach here.